shuffle/move/examples/main/helpers.ts

```diff
@@ -151,16 +191,28 @@ export async function newSignedTransaction(
 }
 
 export function hexToAccountAddress(hex: string): DiemTypes.AccountAddress {
-  if (hex.startsWith("0x")) {
-    hex = hex.slice(2);
-  }
   const senderListTuple: ListTuple<[uint8]> = [];
   for (const entry of hexToBytes(hex)) { // encode as bytes
     senderListTuple.push([entry]);
   }
   return new DiemTypes.AccountAddress(senderListTuple);
 }
 
+// deno-lint-ignore no-explicit-any
+function normalizeScriptFunctionArgs(args: any[]) {
+  return args.map((a) => {
+    if (isString(a) && !a.startsWith("0x")) {
+      return bufferToHex(textEncoder.encode(a));
```

Comment:
Maybe here is the reason we don't have `0x` prefix in the script function arguments. It's fine if you wanted to improve it later.